<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormPrincipal</class>
 <widget class="QMainWindow" name="FormPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>458</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Sistema de Vendas - KAIRÓS</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../img/PGKairos1.png</normaloff>../../img/PGKairos1.png</iconset>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="iconSize">
   <size>
    <width>60</width>
    <height>60</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>83</y>
      <width>1000</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCadastrarCliente">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>150</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>100</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Aqui você pode cadastrar clientes.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="text">
     <string>Clientes</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../img/cadastrar usuario.png</normaloff>../../img/cadastrar usuario.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCadastrarProduto">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>150</width>
      <height>71</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>100</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Aqui você pode listar os clientes cadastrados.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Produtos</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../img/cadastrar produtos.png</normaloff>../../img/cadastrar produtos.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="imgPrincipal">
    <property name="geometry">
     <rect>
      <x>-100</x>
      <y>100</y>
      <width>941</width>
      <height>341</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../img/PGKairosIcone.png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnVenda">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>150</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>100</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Aqui você pode listar os clientes cadastrados.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Vendas</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../img/loja.png</normaloff>../../img/loja.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnContasAReceber">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>150</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>100</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Aqui você pode listar os clientes cadastrados.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Devedores</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../img/contas a receber.png</normaloff>../../img/contas a receber.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRelatorios">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>100</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Aqui você pode listar os clientes cadastrados.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Relatórios</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../img/relatorioVenda.png</normaloff>../../img/relatorioVenda.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>100</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(247, 247, 247);</string>
    </property>
    <property name="text">
     <string>Versão: BETA1.0</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>btnCadastrarCliente</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
